internal/cmd/testdata/success_python.golden

```diff
@@ -278,6 +278,94 @@ class GeneratedClient:
             flag_evaluation_options=flag_evaluation_options,
         )
 
+    def theme_customization(
+        self,
+        evaluation_context: Optional[EvaluationContext] = None,
+        flag_evaluation_options: Optional[FlagEvaluationOptions] = None,
+    ) -> object:
```

Comment:
with the next version (0.8.2) of the Python SDK this can be refined to 
```suggestion
    ) -> typing.Union[Sequence[FlagValueType], typing.Mapping[str, FlagValueType]]:
```